test/shared_utilities/file_reader.jl

```diff
@@ -23,6 +23,16 @@ regrid_dir_temporal =
 isdir(regrid_dir_static) ? nothing : mkpath(regrid_dir_static)
 isdir(regrid_dir_temporal) ? nothing : mkpath(regrid_dir_temporal)
 
+"""
+Set NaN and Missing to zero (GPU compatbile)
+"""
+function replace_nan_missing!(field)
```

Comment:
not a big deal, but should we note that the field needs to be a ClimaCore.Fields.Field?

Reply:
It doesn't really have to be a `ClimaCore.Fields.Field`, the function would work for any struct that has a `parent` (e.g., a `view` of an array).

However, in practice, it would be unwise to use this function for things that are not fields, so I added the annoitation.